libs/langchain/langchain/chains/graph_qa/neptune_cypher.py

```diff
@@ -68,7 +68,7 @@ def use_simple_prompt(llm: BaseLanguageModel) -> bool:
         return True
 
     # Bedrock anthropic
-    if llm.model_id and "anthropic" in llm.model_id:  # type: ignore
+    if hasattr(llm, "model_id") and "anthropic" in llm.model_id:  # type: ignore
```

Comment:
have you considered doing an instance check on  whether the LLM is Anthropic?

```
from langchain.llms.anthropic import Anthropic
return isinstance(llm, Anthropic)
```

It doesn't involve accessing any private attributes, so should be less brittle

Reply:
Thanks for reviewing. I wanted to avoid the imports for the LLMs, for example the check will include Anthropic and ChatAnthropic. Also, for Bedrock and ChatBedrock, we have to check for the model_id value to determine the provider.